### ISSUE TITLE
 By pressing enter or space key no action is performing for “Back to Top” control: A11y_Maow_Back to Top_Functionlaity.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#119]
### GitHub Tags
#A11yTCS; #A11yFunctional; #ChromiumEdge; #Win11; #DesktopWeb; #Maow; #Functionality;

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/catalog/
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:  

1. Hit the URL: https://moaw.dev/catalog/.
2. Tab till back to top button.
5. Verify that by pressing enter or space key action is performing for “Back to Top” control or not.
### Actual Result:  

 By pressing enter or space key no action is performing for “Back to Top” control.
### Expected Results: 
 By pressing enter or space key user should be able to go the top of the page.
### User Impact: 
All the users will face difficulty while accessing the controls after pressing enter or space key no action is preforming.
### Attachment:
![Presentation1](https://github.com/microsoft/moaw/assets/124244478/793cab89-6e7c-4cc9-add7-2be010ad795b)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?
